src/renderer/angular/services/balance-updater.service.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+const { Logger } = require('common/logger/logger');
+const log = new Logger('BalanceUpdaterService');
+function BalanceUpdaterService($rootScope, $interval, Web3Service, RPCService) {
+	'ngInject';
+
+	log.info('BalanceUpdaterService Initialized');
+
+	let txInfoCheckInterval = null;
+	const TX_CHECK_INTERVAL = 1000;
+
+	const updateBalances = async oldBalance => {
+		let walletUpdaterPromise = RPCService.makeCall('getWalletByPublicKey', {
+			publicKey: $rootScope.wallet.getPublicKey()
+		});
+		let tokensUpdaterPromise = RPCService.makeCall('getWalletTokens', {
+			walletId: $rootScope.wallet.id
+		});
+
+		await Promise.all([walletUpdaterPromise, tokensUpdaterPromise]);
+		let currentWallet = await RPCService.makeCall('getCurrentWallet');
+		console.log(oldBalance, currentWallet.balance);
```

Comment:
why not use store directly here with a getCurrentWallet selector? (we want to deprecate RPCService in near future)